### ISSUE TITLE
Piskel duplicating files

### ISSUE THREAD

**Issue by willianholtz** [4ian/GDevelop#2887]
Today I went to review my game folder, and I noticed that there were a lot of files that I wasn't using. And I noticed that my GD .json file was super inflated, at almost 32MB. I know it's not much, but for everything I've written it shouldn't have more than 15~20MB

That's when I went hunting for the BUG, ​​and I realized that if we edit anything in PISKEL, like changing its size or changing it in any way other than its color, it doubles.

https://www.youtube.com/watch?v=d5Y-t2NounM

![resoiureces](https://user-images.githubusercontent.com/23488207/128617314-91900ee2-362d-414e-9c09-b49be2a22e8e.png)

I realized this because I went to clean up my project folder. Now, imagine how many projects are inflated because of this!

At first, I believe this shouldn't happen. If this should actually happen, I believe the app should warn the user.

**Comment by blurymind** [comment]
This is designed so to prevent people from overriding files by mistake.
Gdevelop has a command that automatically removes unused files from a
project which reduces the exported game size

**Comment by Bouh** [comment]
I find that the duplication is not justified.
The user makes a deliberate choice to make a modification on the image, this choice is wanted, we can not keep backups for "in case" the user to change his mind.
The original image must therefore be edited accordingly, the duplication has no place in my opinion. It's not necessary and as you can see it includes a useless file in the project.

**Comment by 4ian** [comment]
I agree we should probably migrate to editing the file directly. Could actually make things much simpler for everyone.
